api/extensions/ext_request_logging.py

```diff
@@ -43,7 +51,30 @@ def _log_request_started(_sender, **_extra):
 
 def _log_request_finished(_sender, response, **_extra):
     """Log the end of a request."""
-    if not logger.isEnabledFor(logging.DEBUG) or response is None:
+    if response is None:
+        return
+
+    # Always emit a compact access line at INFO with trace_id so it can be grepped
+    try:
+        start_ts = getattr(g, "__request_started_ts", None)
+        duration_ms = None
+        if start_ts is not None:
+            duration_ms = round((time.perf_counter() - start_ts) * 1000, 3)
+        req = flask.request
+        trace_id = get_trace_id_from_otel_context() or response.headers.get("X-Trace-Id") or ""
+        logger.info(
+            "%s %s %s %s %s",
+            getattr(req, "method", "-"),
+            getattr(req, "path", "-"),
+            getattr(response, "status_code", "-"),
+            duration_ms if duration_ms is not None else "-",
+            trace_id,
+        )
+    except Exception:
```

Comment:
ditto. 

Besides, there are test cases for `ext_request_logging.py`. I suggest add tests for trace_id related logic and remove `try ... except` block.